smc_trading_agent/v1/routers/trading.py

```
@@ -0,0 +1,157 @@
+        .order_by(Trade.created_at.desc())
+        .limit(100)
+        .all()
+    )
+    return [serialize_trade(trade) for trade in trades]
+
+
+@router.post("/execute", response_model=TradeResponse)
+async def execute_trade(
+    payload: TradeRequest,
+    request: Request,
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)